### ISSUE TITLE
Swipe Up on IPad to switch apps is blocked by bottom bar

### ISSUE THREAD

**Issue by sgruby** [DagAgren/toot#184]
When I swipe up in Toot! To switch apps on my iPad Pro (iOS 12), I get a little up arrow arrow and it tries to create a new Toot. I have to hit the home button to switch apps. I think that the bottom bar is intercepting swipes and not letting the OS get them.

**Comment by sgruby** [comment]
Duplicate of issue #52
